osidb/models/tracker.py

```diff
@@ -387,14 +387,20 @@ def _validate_tracker_duplicate(self, **kwargs):
         validate that there is only one tracker with this update stream associated with each affect
         """
         for affect in self.affects.all():
-            if (
-                affect.trackers.filter(ps_update_stream=self.ps_update_stream).count()
-                > 1
-            ):
+            trackers = affect.trackers.filter(ps_update_stream=self.ps_update_stream)
+            if trackers.count() > 1:
+
                 raise ValidationError(
-                    f"Tracker with the update stream {self.ps_update_stream} "
+                    f"Tracker with the update stream {self.ps_update_stream} ({self.external_system_id}) "
                     "is already associated with the affect "
-                    f"{affect.ps_module}/{affect.ps_component} ({affect.uuid})"
+                    f"{affect.ps_module}/{affect.ps_component} ({affect.uuid}) "
+                    f"by the tracker(s) {', '.join([str(tracker.external_system_id) for tracker in trackers if tracker.uuid != self.uuid])}",
+                    params={
+                        "resolution_steps": "When manually cloning a tracker, please ensure that each affect has a unique update stream, "
+                        "which can be identified in the tracker's title. "
+                        "If the tracker is a duplicate, remove the duplicate tracker. "
+                        "If the tracker is not expected to be a duplicate, please contact the OSIDB team for further assistance."
```

Comment:
I would rather use `Vulnerability Tooling team` as we already used it in bzsync alert, so we should keep that consistent:
https://github.com/RedHatProductSecurity/osidb/blob/master/osidb/models/flaw/flaw.py#L993